pkg/module_manager/module_manager.go

```diff
@@ -709,17 +705,15 @@ func (mm *moduleManager) RefreshEnabledState(logLabels map[string]string) (*Modu
 }
 
 func (mm *moduleManager) GetModule(name string) *Module {
-	module, exist := mm.allModulesByName[name]
-	if exist {
-		return module
-	} else {
-		log.Errorf("Possible bug!!! GetModule: no module '%s' in ModuleManager indexes", name)
-		return nil
+	if mm.modules.Has(name) {
+		return mm.modules.Get(name)
 	}
+	log.Errorf("Possible bug!!! GetModule: no module '%s' in ModuleManager indexes", name)
```

Comment:
It's not a bug, just no such module. It is fine, CALM DOWN AND DON'T PANIC!!!

```
deckhouse-controller module values my-inexisting-module
```

Reply:
Good point, yeah. The caller should check for nil.

Reply:
it's more about the exclamation ""Possible bug!!!" :)

Reply:
Done